repository/starkpill/metadata.json

```
@@ -0,0 +1,19 @@
+{
+  "id": "starkpill",
+  "displayName": "Starkpill",
+  "host": "starkpill.me",
```

```suggestion
  "host": "www.starkpill.me",
```

this is the actual host calculated from the dapp request 👆